lib/purescript-cst/src/Language/PureScript/CST/Parser.y

```diff
@@ -123,6 +123,7 @@ import Language.PureScript.PSString (PSString)
   'then'             { SourceToken _ (TokLowerName [] "then") }
   'true'             { SourceToken _ (TokLowerName [] "true") }
   'type'             { SourceToken _ (TokLowerName [] "type") }
+  'via'              { SourceToken _ (TokLowerName [] "via") }
```

Comment:
Since this is a contextual keyword, you will need to make sure to add it `ident`, `qualIdent`, and `label`.

Reply:
Good catch! Sorry about that, I added a few test cases to cover this.